target/linux/ath79/image/generic.mk

```diff
@@ -529,6 +529,15 @@ define Device/comfast_cf-wr752ac-v1
 endef
 TARGET_DEVICES += comfast_cf-wr752ac-v1
 
+define Device/comfast_cf-wr610n
```

Comment:
Please take care of alphabetic sorting. No v1/v2?

Reply:
ok, only this version.

> No setup for lan/wan LEDs?

yes, ALL PORT used default

Reply:
If the WAN/LAN LEDs work automatically without any specific setup, then you probably used the wrong gpio numbers.

Reply:
Still need to fix alphabetic sorting here.